arangod/Agency/AgencyComm.cpp

```diff
@@ -542,7 +542,7 @@ void AgencyCommHelper::initialize(std::string const& prefix) {
 
 void AgencyCommHelper::shutdown() {}
 
-std::string AgencyCommHelper::path() { return PREFIX; }
+const std::string& AgencyCommHelper::path() noexcept { return PREFIX; }
```

Comment:
This is obviously valid, but in 99% of the rest of the code we are doing:
```suggestion
std::string const& AgencyCommHelper::path() noexcept { return PREFIX; }
```